libs/expo/betterangels/src/lib/screens/ClientHMIS/tabs/ClientLocationsHmisView/ClientLocationsHmisViewModal.tsx

```diff
@@ -0,0 +1,94 @@
+import { Colors, Spacings } from '@monorepo/expo/shared/static';
```

Comment:
almost line for line copy of `libs/expo/betterangels/src/lib/screens/Client/Locations/InteractionLocationsModal.tsx` can we reuse?